pkgs/development/tools/earthly/default.nix

```diff
@@ -35,7 +35,7 @@ buildGoModule rec {
     description = "Build automation for the container era";
     homepage = "https://earthly.dev/";
     changelog = "https://github.com/earthly/earthly/releases/tag/v${version}";
-    license = licenses.bsl11;
-    maintainers = with maintainers; [ mdsp ];
+    license = with licenses; [ bsl11 mpl20 ];
```

Comment:
i think **this** version is not mpl20 **now**, so it should not be listed here

Reply:
hmmm
as i understand all the code was allways licensed by these two licenses

actually, some parts of their code are licensed by mpl20, however this passed unnoticed in my initial PR

Reply:
https://github.com/earthly/earthly says at the end of the README "Earthly is licensed under the Business Source License 1.1. See licenses/BSL for more information."

Reply:
https://github.com/earthly/earthly/blob/main/LICENSE does also only mention BSL

Reply:
yeah! that's rigtht, however in  the LICENSE file it's says that "unless otherwise noted at the beginning of the file." 

kinda confusing haha
but ok, I'll keep only bsl

Reply:
I also saw that but couldn't find anything mspl licensed while quickly looking. Also I think this only represents the primary license as dependencies probably have lots of other licenses but I actually don't know that the intended meaning of the licenses attribute is.

Reply:
I also saw that but couldn't find anything mspl licensed while quickly looking. Also I think this only represents the primary license as dependencies probably have lots of other licenses but I actually don't know that the intended meaning of the licenses attribute is.

Reply:
if i understand it correctly this unfree license will become open source (Mozilla Public License Version 2.0) on 2024-04-01 for Earthly v0.5

https://github.com/earthly/earthly/blob/main/licenses/BSL

so you could add  it at that point, but there will be a new version that will become open source later in the future

so in practice, this is not open source and you shouldn't use and support it. but that's just my opinion

Reply:
oh, that's makes much more sense. now i got it!